HOWTOCONTRIBUTE.rst

```diff
@@ -160,6 +160,39 @@ Preparing Pull Requests
 
    This command will run tests via the "pytest" tool against Python 3.6.
 
+#. Running the performance test suite
+
+Performance matters and it is worth considering whether your code has introduced
+performance regressions. *climpred* is starting to write a suite of benchmarking tests
+using `asv <https://asv.readthedocs.io/en/stable/>`__
+to enable easy monitoring of the performance of critical *climpred* operations.
+These benchmarks are all found in the ``climpred/asv_bench`` directory.
+
+If you need to run a benchmark, change your directory to ``asv_bench/`` and run::
+
+    $ asv continuous -f 1.1 upstream/master HEAD
+
+You can replace ``HEAD`` with the name of the branch you are working on,
+and report benchmarks that changed by more than 10%.
+The command uses ``conda`` by default for creating the benchmark
+environments.
+
+Running the full benchmark suite can take up to half an hour and use up a few GBs of
+RAM. Usually it is sufficient to paste only a subset of the results into the pull
+request to show that the committed changes do not cause unexpected performance
+regressions.  You can run specific benchmarks using the ``-b`` flag, which
+takes a regular expression.  For example, this will only run tests from a
+``climpred/asv_bench/benchmarks/benchmarks_perfect_model.py`` file::
```

Comment:
Similarly I would drop the front ``climpred``. I get what you mean here but references are typically to the base directory. Then ``climpred`` means the folder with code in it.